### ISSUE TITLE
Realm Browser 3 doesn't connect to older ROS

### ISSUE THREAD

**Issue by agersoncgps** [realm/realm-browser-osx#361]
When I try to connect to ROS v1.8.3 with Realm Browser 3.0 I get this:

<img width="681" alt="screenshot 2017-10-22 17 34 46" src="https://user-images.githubusercontent.com/336441/31866575-a7644924-b74f-11e7-8ddd-f0497bb83949.png">

**Comment by hslatman** [comment]
The new Realm Browser does not seem to be backwards compatible with ROS v1.8.3, at least, that's what I understand from the changes in the RealmSwift pod 3.0.0, which is used in this project.

You can download an older version of the Realm Browser via [https://github.com/realm/realm-browser-osx/releases](https://github.com/realm/realm-browser-osx/releases).

**Comment by agersoncgps** [comment]
Ok, there should be a message that pops up when you try to connect to an older version. The Apple App store aggressively pushes the update and then leaves the users confused and unable to connect.

**Comment by hslatman** [comment]
Yes, that's something that the Realm team could've warned for or could've implemented. I can imagine that implementing both versions of the pods was problematic. Honestly, I thought they would be abandoning the Realm Browser all together because of the new Realm Studio.

**Comment by bmunkholm** [comment]
Yes, that's correct. @agersoncgps I would also encourage to use the new Realm Studio (https://realm.io/docs/realm-object-server/latest/#realm-studio) going forward unless you depend on import/export currently.
